backend/de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/github/process/RegisterIssuesWebhookProcess.java

```diff
@@ -54,7 +56,11 @@ public ProcessPreconditionsResolution checkPreconditionsApplicable(final @NonNul
 	@Override
 	protected String doIt() throws Exception
 	{
-		githubService.createSyncIssuesWebhook(ExternalProjectReferenceId.ofRepoId(getRecord_ID()));
+		final ExternalProjectReferenceId externalProjectReferenceId = ExternalProjectReferenceId.ofRepoId(getRecord_ID());
+
+		Loggables.withLogger(log, Level.DEBUG).addLog("Invoking RegisterIssuesWebhookProcess for ExternalProjectReferenceId: {}", externalProjectReferenceId.getRepoId());
```

Comment:
JavaProcess class already have `addLog` methods.
I am wondering why not using them instead of `Loggables.withLogger(log, Level.DEBUG)`?

also, why using `externalProjectReferenceId.getRepoId()` instead of simply `externalProjectReferenceId`?

Reply:
changed to use `addLog` from `JavaProcess` and dropped `.getRepoId()`. thx for noticing